libgnucash/engine/iso-4217-currencies.xml

```diff
@@ -2414,6 +2414,7 @@
   local-symbol=""
 />
 <!-- "SZL" - "Lilangeni"
+  2019-02-16 : The plural "E" is the common local-symbol; Bug 797105
```

Comment:
No date here, please. The dates in the comments mean 'date of replacement' followed by new currency and frozen rate.

Reply:
Thanks for clarifying. Fixed.